packages/bootleg-app/src/TraderComponent.js

```diff
@@ -1,82 +1,137 @@
+  ContractForm
+} from 'drizzle-react-components';
+import { Button, Form, Input, Card } from 'rimble-ui';
+
+const tokenPurchasePrice = 1000000;
 
 const TraderComponent = ({ accounts, drizzleStatus }) => (
   <div className="App">
-    <div>
-      <h1>Drizzle Examples</h1>
-      <p>Examples of how to get started with Drizzle in various situations.</p>
-    </div>
-    <div className="section">
-      <h2>The Token</h2>
-      <ContractData contract="BootlegTraderApp" method="bootlegTokenId" />
-      <h2>Token URI</h2>
-      <ContractData contract="BootlegTraderApp" method="getTokenURI" />
-      <h2>Token Owner</h2>
-      <ContractData contract="BootlegTraderApp" method="getOwner" />
-      <h2>Your Balance</h2>
-      <ContractData contract="BootlegTraderApp" method="getBalance" />
-      <h2>Active Account</h2>
-      {JSON.stringify(accounts)}
-      <h2>Purchase</h2>
-      <ContractForm
-        contract="BootlegTraderApp"
-        method="purchase"
-        render={({
-          inputs,
-          inputTypes,
-          state,
-          handleInputChange,
-          handleSubmit
-        }) => (
-          <Form onSubmit={handleSubmit}>
-            <Button
-              key="submit"
-              type="button"
-              onClick={handleSubmit}
-              size={'medium'}
-            >
-              Purchase
-            </Button>
-          </Form>
-        )}
-      />
-      <h2>List for sale</h2>
-      <ContractForm
-        contract="BootlegTraderApp"
-        method="setTokenPrice"
-        render={({
-          inputs,
-          inputTypes,
-          state,
-          handleInputChange,
-          handleSubmit
-        }) => (
-          <Form onSubmit={handleSubmit}>
-            {inputs.map((input, index) => (
-              <Input
-                style={{ fontSize: 30 }}
-                key={input.name}
-                type={inputTypes[index]}
-                name={input.name}
-                value={state[input.name]}
-                placeholder={input.name}
-                onChange={handleInputChange}
-              />
-            ))}
-            <Button
-              key="submit"
-              type="button"
-              onClick={handleSubmit}
-              size={'medium'}
-            >
-              List for sale
-            </Button>
-          </Form>
+    <Card fontSize={4}>
+      <span>You are logged in with the following account:</span>
+      <AccountData
+        accountIndex="0"
+        units="ether"
+        precision="3"
+        render={({ address, balance, units }) => (
+          <div>
+            <div>
+              Address: <span style={{ color: 'red' }}>{address}</span>
+            </div>
+            <div>
+              Balance: <span style={{ color: 'red' }}>{balance}</span> {units}
+            </div>
+          </div>
         )}
       />
+    </Card>
+    <div className="section" style={{ padding: '1em' }}>
+      <h1>The Bootleg Token Trader</h1>
+      <div
```

Comment:
Rather than using a bunch of inline-styles, try using flexbox via rimbles `Flex` and `Box` components https://consensys.github.io/rimble-ui/?path=/story/components-layout-flex--documentation